python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -44,18 +44,19 @@ def __str__(self):
             err = ''
         return "XMLFileError: %s %s" % (self.message, err)
 
-def _raw_to_file(j, fobj=None, ignore_subs=''):
+def _raw_to_file(j, fobj=None, ignore_subs=[]):
     vstreamer = VStreamer(out=fobj, selection=ignore_subs)
     vstreamer.begin_root()
     stripProxy(j).accept(vstreamer)
     vstreamer.end_root()
 
-def to_file(j, fobj=None, ignore_subs=''):
+def to_file(j, fobj=None, ignore_subs=[]):
     #used to debug write problems - rcurrie
     #_raw_to_file(j, fobj, ignore_subs)
     #return
+    _ignore_subs = [ignore_subs] if not isinstance(ignore_subs, list) else ignore_subs
```

Comment:
Yes ```GangaRepositoryXML``` to (among other things I think) not write all of the subjobs into the master job ```data```